collated/main/A0121533W.md

```diff
@@ -0,0 +1,797 @@
+# A0121533W
+###### \java\tars\commons\util\DateTimeUtil.java
+``` java
+     */
+    public static boolean isWithinWeek(LocalDateTime endDateTime) {
+        if (endDateTime == null) {
+            return false;
+        } else {
+            LocalDateTime today = LocalDateTime.now();
+            LocalDateTime startThisWeek = today.with(TemporalAdjusters.previous(DayOfWeek.SUNDAY));
```

Comment:
coding standard violation; Avoid lines longer than 80 chars. When a line is too long, it requires horizontal scrolling, that slows down the reader/reviewer.